### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by jessica-yauney**

### CODE REVIEW

File: guide/src/config/endpoints-section.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The `endpoints` section declares what HTTP endpoints will be called during a tes
...
+  Tags are a series of key/value pairs used to distinguish each endpoint. Tags can be used to include certain endpoints in a [`try`] run(../cli.md#Command-line-options), and also make it possible for a single endpoint to have its results statistics aggregated in multiple groups. Because tag values are [templates](./common-types.md#templates) only tags which can be resolved statically at the beginning of a test can be used with the `include` flag of a `try` run. A reference to a provider can cause a single endpoint to have multiple groups of tags. Each one of these groups will have its own statistics in the results. For example if an endpoint had the following tags:
```
Comment (tkmcmaster) [inline guide/src/config/endpoints-section.md:42]: Did you mean to change this? The link is currently broken with this change. I think you may have wanted:
```
[`try`](../cli.md#Command-line-options) run
```

**Review by tkmcmaster — CHANGES_REQUESTED** [review]
Thanks for fixing the link @jessica-yauney. You can test it out locally by using the `build-guild.sh` and `serve-guild.sh` scripts in the ./guide/ directory. You'll need to have `rust`, `wasm-pack`, and `mdbook` installed. The instructions for installing the later two are in the .sh script files at the top.

File: guide/src/config/endpoints-section.md (hunk 1 above)
Comment (jessica-yauney) [reply guide/src/config/endpoints-section.md]: Yes- sorry

**Review by tkmcmaster — APPROVED** [review]
Looks good Jessica. Thanks for catching and fixing this.

**Pull request merged**